lightning/src/routing/network_graph.rs

```diff
@@ -1085,6 +1085,63 @@ impl NetworkGraph {
 		self.update_channel_from_unsigned_announcement_intern(msg, None, chain_access)
 	}
 
+	/// Update channel from partial announcement data received via rapid gossip
+	///
+	/// `timestamp: 64`: Timestamp the announcement was originally received.
+	///
+	/// All other parameters as used in `UnsignedChannelAnnouncement` fields.
+	pub fn update_channel_from_announcement_fragments(&self, short_channel_id: u64, timestamp: u64, features: ChannelFeatures, node_id_1: PublicKey, node_id_2: PublicKey) -> Result<(), LightningError>{
```

Comment:
Lets just call this `add_channel` or something? This name is much more specific than it need be.

Reply:
I came up with a different name, but `add_channel` is definitely not clear given the context of when this function should be called.